ambry-replication/src/test/java/com/github/ambry/replication/BlobIdTransformerTest.java

```diff
@@ -527,7 +527,7 @@ private Message buildMessage(String blobIdString, Class clazz, boolean hasEncryp
       MessageInfo messageInfo;
       BlobProperties blobProperties =
           new BlobProperties(blobPropertiesSize, "serviceId", "ownerId", "contentType", false, 0, 0,
-              blobId.getAccountId(), blobId.getContainerId(), hasEncryption, null);
+              blobId.getAccountId(), blobId.getContainerId(), hasEncryption, null, "gzip", "filename");
```

Comment:
minor: is there a unit test to verify these two properties in BlobIdTransformerTest?

Reply:
Will add the test cases once the serialize part is done.